README.md

```diff
@@ -70,38 +67,66 @@ To get a local copy up and running follow these simple steps.
 
 ### Prerequisites
 
-This is an example of how to list things you need to use the software and how to install them.
-* npm
+* python
 ```sh
```

Comment:
Well, this seems Debian specific, you can add like a few more maybe?
Or just leave it as
- Python 3.x

Reply:
Also, add the new files that have been added to [csivitu/template](https://github.com/csivitu/template)